packages/form-js-editor/src/render/components/FormEditor.js

```diff
@@ -65,17 +65,27 @@ function ContextPad(props) {
   );
 }
 
-function Empty() { return null; }
-
-function EmptyRoot(props) {
-  return <div class="fjs-empty-editor">
-    <div class="fjs-empty-editor-card">
-      <EmptyFormIcon />
-      <h2>Build your form</h2>
-      <span>Drag and drop components here to start designing.</span>
-      <span>Use the preview window to test your form.</span>
-    </div>
-  </div>;
+function Empty(props) {
+  if (props.field.type === 'default') {
+    return <div class="fjs-empty-editor">
+      <div class="fjs-empty-editor-card">
+        <EmptyFormIcon />
+        <h2>Build your form</h2>
+        <span>Drag and drop components here to start designing.</span>
+        <span>Use the preview window to test your form.</span>
+      </div>
+    </div>;
+  }
+
+  if (props.field.type === 'group') {
+    return <div class="fjs-empty-component"><span>Drag and drop components here.</span></div>;
+  }
+
+  if (props.field.type === 'dynamiclist') {
+    return <div class="fjs-empty-component"><span>Drag and drop components here <br /> to create a repeatable list item.</span></div>;
+  }
```

Comment:
The drop container is slighty overlapping the placeholder.

![image](https://github.com/bpmn-io/form-js/assets/9433996/4a2909f7-3025-411e-92b3-5d56dd56fa34)

I wonder whether we should hide the placeholder during drag & drop @volodymyr-melnykc